package.json

```diff
@@ -4,25 +4,41 @@
   "description": "",
   "main": "build/index.js",
   "types": "build/index.d.ts",
+  "keywords": [
+    "config",
+    "typescript",
+    "node-config",
+    "environment-variables",
+    "env",
+    "environment",
+    "settings"
+  ],
+  "author": "Stephen Haberman",
+  "license": "MIT",
+  "homepage": "https://github.com/stephenh/ts-app-env/#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stephenh/ts-app-env.git"
+  },
+  "bugs": "https://github.com/stephenh/ts-app-env/issues",
   "scripts": {
     "test": "jest",
-    "build": "tsc"
+    "build": "tsc",
+    "lint": "eslint src --ext .ts"
   },
-  "keywords": [],
-  "author": "Stephen Haberman",
-  "license": "MIT",
-  "devDependencies": {
-    "@types/jest": "^23.3.2",
-    "@types/node": "^10.7.0",
-    "jest": "^23.6.0",
-    "nodemon": "^1.18.3",
-    "prettier": "^1.14.3",
-    "ts-jest": "^23.10.2",
-    "ts-node": "^7.0.1",
-    "tslint": "^5.11.0",
-    "tslint-config-prettier": "^1.15.0",
-    "tslint-immutable": "^4.7.0",
-    "typescript": "^3.0.1"
+  "engines": {
+    "node": ">=12"
```

Comment:
I could also see only supporting 14+ since 12 is only good until April. I figure that this is simple enough that we can probably safely support 12.